src/dropdown/list/listdropdownview.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import DropdownView from '/ckeditor5/ui/dropdown/dropdownview.js';
+
+/**
+ * The basic list dropdown view class.
+ *
+ * @memberOf ui.dropdown
+ * @extends ui.View
+ */
+
+export default class ListDropdownView extends DropdownView {
+	constructor( model ) {
+		super( model );
+
+		this.listenTo( this.model, 'change:isOn', ( evt, name, value ) => {
```

Comment:
No. Because it would mean that as long as editor is present in the web page, ANY click in that web page, wherever it happens, would trigger this listener and logic within it.